conda_forge_tick/settings.py

```diff
@@ -0,0 +1,39 @@
+import os
```

Comment:
Good job with the name docstrings! A module docstring here would be useful to inform of the purpose of this module and the different design decisions you took:

- Each environment variable _name_ is _also_ given a Python name so it can be reused in other parts of the library.
- There's no validation of the passed value


Also I think you could add type hints to the variables. Most are str but I'm seeing bools and floats too.

Reply:
👍 , done in 318975ce96363bf6c6e6a163bc19165cbc63418a